README.md

```diff
@@ -29,6 +29,7 @@ Indaba puts your stakeholder and expert network at your fingertips. It converts
 - pgAdmin (optional)
 - memcached
 - nginx (for server deployment)
+- Docker
```

Comment:
I know we want to get this merge in asap but should we maybe update this README with this file:
https://paper.dropbox.com/doc/Indaba-Set-up-for-new-developers-Mac-5NzKPSGxbHOUXomHFaEKy

Reply:
@Perry5 I have doubts because that only covers it for Mac. How about we push this as is and revisit that?

Reply:
👍